01week/datatypes.js

```diff
@@ -0,0 +1,66 @@
+`use strict`
+
+//returns current date and time
+//make a today var, which is assigned a new date instance
+//make new var to store the today var and toDateString method
+//use var today with toTimeString method
+
+var today = new Date(1985, 11, 1);
+console.log(today.toDateString());
+
+//create a function that will supply the resulting string
+//create a var to convert.
+//input a number into var convert
+//use convert and toString() method to make the number a string
+
+function toNumber() {
+  var num = 13
+  var n = console.log(num.toString)
```

Comment:
This n is redundant, and the purpose of the function is to take in any number and convert it to a string. This function console logs 13 to a string. The return statement should look like this `return num.toString`.